kebechet/managers/thoth_advise/README.rst

```diff
@@ -0,0 +1,43 @@
+Kebechet Advise-Results Manager
+-----------------------
+This manager is automatically triggered after the `ThothAdviseManager` finishes.  It will take the results and create pull requests and issues based on the info gathered
```

Comment:
This looks like an implementation detail. As a user of Thoth+Kebechet I just want to know the purpose of this manager - managing my project dependencies.